opslevel/resource_opslevel_check_base.go

```diff
@@ -0,0 +1,136 @@
+package opslevel
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
+	"github.com/hashicorp/terraform-plugin-framework-validators/boolvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/opslevel/opslevel-go/v2024"
+	"time"
+)
+
+var checkBaseAttributes = map[string]schema.Attribute{
+	"category": schema.StringAttribute{
+		Description: "The id of the category the check belongs to.",
+		Required:    true,
+		Validators:  []validator.String{IdStringValidator()},
+	},
+	"description": schema.StringAttribute{
```

Comment:
I noticed `description` isn't part of the [check_manual](https://registry.terraform.io/providers/OpsLevel/opslevel/latest/docs/resources/check_manual) schema, or any of the other `opslevel_check_*` resources

Reply:
switched to computed